### PR TITLE
Resolved conflicts

### PR DESCRIPTION
**Pull request by raulovidiu**

### CODE REVIEW

File: spec.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const faker = require('faker');
+const CreateBlogPost = require('./page-objects/create-blog-post.pageObject.js');
```
Comment (wlsf82) [inline spec.js]: you missed changing the page object file here.

File: spec.js [hunk 2]
```diff
@@ -23,17 +24,36 @@ describe('Choko - Create account', () => {
...
+  });
 
+  it('Create blog post without permission, passing 4 new lines and 6 paragraphs in the body', () => {
     browser.get('create/blog');
 
-    element(by.id('element-type-blog-name')).sendKeys(randomName);
-    element(by.id('element-type-blog-title')).sendKeys(randomTitle);
-    element(by.className('note-editable')).sendKeys(randomText);
-    element(by.id('element-type-blog-submit')).click();
+    createBlogPost.name.sendKeys(ramdonName);
```
Comment (wlsf82) [inline spec.js]: ramdonName is not defined. It should be randomName.

**Comment by raulovidiu** [comment]
Integrated the change. Thank you.

**Comment by wlsf82** [comment]
🍰

**Pull request merged**